website/content/docs/other-specifications/volume/host.mdx

```diff
@@ -0,0 +1,285 @@
+---
+layout: docs
+page_title: Host Volume Specification
+description: Learn about the Volume specification used to create and register dynamic host volumes to Nomad.
+---
+
+# Dynamic Host Volume Specification
+
+The Nomad volume specification defines the schema for creating and registering
+dynamic host volumes using the [`volume create`][] and [`volume register`][]
+commands and the [`PUT /v1/volume/host/create`][api_volume_create] and [`PUT
+/v1/volume/host/register`][api_volume_register] API endpoints.
+
+Some attributes are only be supported by specific operation, while others may
+have a different meaning for each action, so read the documentation for each
+attribute carefully. The section [Differences Between Create and
+Register](#differences-between-create-and-register) provides a summary of the
+differences.
+
+The file may be provided as either HCL or JSON to the commands and as JSON to
+the API.
+
+## Volume Specification Parameters
+
+- `capacity` `(string: <optional>)` - The size of a volume in bytes. either the
+  physical size of a disk or a quota, depending on the plugin. This field must
+  be between the `capacity_min` and `capacity_max` values unless they are
+  omitted. Accepts human-friendly suffixes such as `"100GiB"`. Only supported
+  for volume registration.
+
+- `capacity_min` `(string: <optional>)` - Option for requesting a minimum
+  capacity, in bytes. The capacity of a volume may be the physical size of a
+  disk, or a quota, depending on the plugin. The specific size of the resulting
+  volume will be somewhere between `capacity_min` and `capacity_max`; the exact
+  behavior is up to the plugin. If you want to specify an exact size, set
+  `capacity_min` and `capacity_max` to the same value. Accepts human-friendly
+  suffixes such as `"100GiB"`. Plugins that cannot restrict the size of volumes
+  may ignore this field.
+
+- `capacity_max` `(string: <optional>)` - Option for requesting a maximum
+  capacity, in bytes. The capacity of a volume may be the physical size of a
+  disk, or a quota, depending on the plugin. The specific size of the resulting
+  volume will be somewhere between `capacity_min` and `capacity_max`; the exact
+  behavior is up to the plugin. If you want to specify an exact size, set
+  `capacity_min` and `capacity_max` to the same value. Accepts human-friendly
+  suffixes such as `"100GiB"`. Plugins that cannot restrict the size of volumes
+  may ignore this field.
+
+- `capability` <code>([Capability][capability]: &lt;required&gt;)</code> -
+  Option for validating the capability of a volume.
```

Comment:
All DHV capabilities are `single-node-*`, right? I don't see that mentioned here in the DHV spec page, nor the `capability` page.

Reply:
I missed that, yeah. I'll copy over what's done in https://github.com/hashicorp/nomad/pull/24828